### ISSUE TITLE
`Main menu > View > Collapse Level > N` works only for the first level for Python scripts

### ISSUE THREAD

**Issue by 8day** [notepad-plus-plus/notepad-plus-plus#4633]
### Steps to Reproduce the Issue
1. open Python file with multiple levels of indentation;
2. go to `Main menu > View > Collapse Level >  2`.

### Expected Behavior
Blocks at 2nd level should've been collapsed.

### Actual Behavior
Nothing happens.

### Debug Information
```
Notepad++ v7.5.7   (64-bit)
Build time : Jun 29 2018 - 00:53:58
Path : c:\path\npp\notepad++.exe
Admin mode : OFF
Local Conf mode : ON
OS : Windows 7 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll 
```
Folding markers are present, and you can fold manually, but not automatically.

**Comment by SinghRajenM** [comment]
Please provide debug info.....

However, it is working fine for me.
![image](https://user-images.githubusercontent.com/14791461/42328763-c2d21446-808c-11e8-9ac6-c1c5f44af7e5.png)

**Comment by 8day** [comment]
Sorry, I was mistaken: this feature doesn't work only in Python. See updates in first post.

**Comment by cmeriaux** [comment]
In the python lexer, a level equals the total number of space. Example with a 4 spaces indentation, the 1rst level of indentation equals de 4th level of folding.
I've reported the issue to the python lexer author but I refused to fixe it because the indentation type can be mixed on the same file.  It's strange but possible.

**Comment by 8day** [comment]
It seems that you are only partially, but mostly right. It seems that that lexer ignores `\t` and detects indentation only for ` `, which makes it even more horrible. IMO `\t` with configurable 'width' would've been much better, almost perfect solution for this issue.

**Comment by cmeriaux** [comment]
@8day here is a thread about that with the lexer author
https://sourceforge.net/p/scintilla/bugs/1834/

**Comment by zufuliu** [comment]
This was fixed by PR #5131.
